app/dashboard/templates/profiles/profile.html

```diff
@@ -121,75 +125,85 @@ <h1 class="profile-header__handle">
-                      {% if scoreboard_position_contributor %}
-                        <b>#{{ scoreboard_position_contributor }}</b>
-                      {% else %}
-                        <b>&nbsp;</b>
-                      {% endif %}
-                      {% trans "Contributor" %}
-                    </div>
-                    <div class="card-body">
-                      <ul>
-                        <li><b>{{ count_bounties_completed }}</b> {% trans "bounties completed" %}</li>
-                        <li><b>{{ sum_eth_collected|floatformat:2 }} ETH</b> {% trans "collected" %}</li>
-                        {% if no_times_been_removed %}
-                          <li> - {% trans "removed from" %} <b>{{ no_times_been_removed }}</b> {% trans "bounties" %}</li>
-                        {% endif %}
-                        <li class="works_with_list">
-                          {% if works_with_collected|length %}
-                              <span class="font-body"><i class="fa fa-code"></i> Contributes To:</span>
-                              {% for profile, num_times in works_with_collected.items %}
-                                {% if forloop.counter < 6 %}
-                                  <a href="{% url 'profile' profile %} ">
-                                    <img src='{% avatar_url profile %}' title="<div class='tooltip-info tooltip-xs'>{{ profile }}: {{ num_times }} times</div>">
-                                  </a>
-                                {% endif %}
-                              {% endfor %}
-                              {% if works_with_collected|length > 5 %}
-                                <div href="" class="plus-img" data-container="body" data-toggle="popover" data-template='<div class="popover big-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>' data-html="true" data-placement="bottom" data-trigger="hover click" data-content='{% for profile, num_times in works_with_collected.items %}<a href="{% url "profile" profile %}"><img class="rounded-circle m-1" width="30" height="30" title="{{profile}}: {{ num_times }} times" src="{% avatar_url profile %}"/></a>{% endfor %}'><span>+{{works_with_collected|length|add:"-5"}}</span></div>
-                              {% endif %}
-                          {% endif %}
-                        </li>
-                      </ul>
-                    </div>
+              <div class="col-12 col-lg-6">
+                <div class="row">
+                  {% if is_my_profile%}
+                  <div class="col-12 text-right">
+                    <button type="button" id="update-profile" class="btn btn-block btn-sm btn-primary animate-verify">{% trans 'Update Profile' %}</button>
+                    <button type="button" id="cancel-profile-update" class="btn btn-block btn-sm btn-outline-gc-purple animate-verify">{% trans 'Cancel' %}</button>
```

Comment:
Instead of "update profile" can we put something like "change header" ?

Reply:
I think should reflect better the user action with that sentence.

Reply:
Making the change now.